region/crateria/west.json

```diff
@@ -2165,6 +2165,33 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [ "h_canBombThings" ]
+                },
+                {
+                  "name": "Speedball",
+                  "notable": false,
+                  "requires": [
+                    "canMockball",
+                    "canCarefulJump",
+                    {"or": [
+                      {"and": [
+                        "canSlowShortCharge",
+                        {"canComeInCharged": {
+                          "framesRemaining": 180,
+                          "fromNode": 2,
+                          "inRoomPath": [2,3],
+                          "shinesparkFrames": 0,
+                          "unusedTiles": 9
+                        }}
+                      ]},
+                      {"canComeInCharged": {
```

Comment:
Would this one, with full speed, require a tricky jump? (and possibly on some of the others without short charge?)